pkg/core/contracts/pools/stable/StableMath.sol

```diff
@@ -357,34 +355,40 @@ contract StableMath {
         uint256 bptTotalSupply,
         uint256 swapFeePercentage
     ) internal pure returns (uint256) {
+        // Token out, so we round down overall.
+
         // Get the current invariant
-        uint256 currentInvariant = _calculateInvariant(amp, balances);
+        uint256 currentInvariant = _calculateInvariant(amp, balances, true);
```

Comment:
If we're rounding down, why is the invariant rounded up? It'd be good to have a short comment about this